valohai_yaml/objs/parameter.py

```diff
@@ -106,12 +106,18 @@ def _validate_type(self, value: ValueAtomType, errors: List[str]) -> ValueAtomTy
             try:
                 value = int(str(value), 10)
             except ValueError:
-                errors.append(f'{value} is not an integer')
+                if not value:
+                    errors.append('No value supplied')
```

Comment:
Hmm... What happens here if you pass in `False` (boolean, not a string)? I think it'll say "No value supplied", but that sounds incorrect...

Reply:
Good idea, I've tightened up the logic and added some more tests.

I also made the call to properly complain about validating Nones instead of tossing IndexError. Github actions seem to be feeling bad so let's try those again tomorrow.